src/pages/tutorials/build-an-nft-marketplace/part-2.md

```diff
@@ -851,7 +856,7 @@ As you are connected with the default administrator you can see your own unique
 - Click on **Sell bottle** sub menu
 - You are now the owner of this bottle, you can resell it at your own price, etc ...
 
```

Comment:
"As you are connected with the default administrator you can see your own unique offer on the market" => "While you are connected with the administrator account you can see the offer in the marketplace page."

Reply:
The list should be ordered steps instead of an unordered list.